R/wrappers.R

```diff
@@ -7,6 +7,7 @@ setcoalesce = function(...) .Call(Ccoalesce, list(...), TRUE)
 
 fifelse = function(test, yes, no, na=NA) .Call(CfifelseR, test, yes, no, na)
 fcase   = function(..., default=NA) .Call(CfcaseR, default, parent.frame(), as.list(substitute(list(...)))[-1L])
+topn    = function(vec, n=6L, decreasing=FALSE) .Call(CtopnR, vec, n, decreasing)
```

Comment:
We also have `na.last` argument for `data.table:::forder`, we should include that here if we really want to replace `order(x)[1:n]`

Reply:
`na` are always last, I did not look at `forder` but the behaviour of `topn` should be similar to `order` when it comes to `NA`.